tools/chunk/src/main/scala/org/allenai/nlpstack/chunk/OpenNlpChunker.scala

```diff
@@ -18,6 +18,23 @@ class OpenNlpChunker extends Chunker {
     val chunks = chunker.chunk(tokens.map(_.string).toArray, fixedTokens.map(_.postag).toArray)
     (tokens zip chunks) map { case (token, chunk) => ChunkedToken(token, chunk) }
   }
+
+  //Open NLP runs into random nullPointerExceptions. The following is a workaround that reduces the
+  // probability
+  // of running into errors
+  def chunkPostagged(tokens: Seq[PostaggedToken]): Seq[ChunkedToken] = {
+    var tries = 3
+    while(tries > 0){
+      tries -= 1
+      val ret = try{
+        chunkPostaggedOriginal(tokens)
+      } catch{
+        case e: Throwable => null
+      }
+      if(ret != null)  return ret
+    }
+    return tokens.map{case a: PostaggedToken => ChunkedToken.apply(a,"ERROR")}
```

Comment:
1. But exceptions would take us where we started I guess. I think we should work on a solid solution.